util/which-key/which-key.lisp

```diff
@@ -0,0 +1,45 @@
+;;;; which-key.lisp
+
+(in-package #:which-key)
+
+(export '(get-kmaps-at-key-seq key-press-hook replace-hook))
```

Comment:
Although this idiom is used throughout the StumpWM codebase, it is an archaism in CL. Consider moving this to the export clause in the package definition

ej.
```lips
(defpackage #:which-key
    (:use #:cl)
    (:export #:get-kmaps-at-key
                 #:key-press-hook
                 ...)
```